File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2203.py
from math import sqrt

while True:
    try:
        ls = input().split(' ')
        xf = int(ls[0])
        yf = int(ls[1])
        xi = int(ls[2])
        yi = int(ls[3])
        vi = int(ls[4])
        r1 = int(ls[5])
        r2 = int(ls[6])
        x = (xf-xi)*(xf-xi)
        y = (yf-yi)*(yf-yi)
        if(sqrt(x+y)+vi*1.5>r1+r2): print("N")
        else: print("Y")
    except:
        break

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2203.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split('\n')

while(linha = lines.shift()){
    ls = linha.split(' ')
    
    xf = parseInt(ls[0])
    yf = parseInt(ls[1])
    xi = parseInt(ls[2])
    yi = parseInt(ls[3])
    vi = parseInt(ls[4])
    r1 = parseInt(ls[5])
    r2 = parseInt(ls[6])
    x = (xf-xi)*(xf-xi)
    y=(yf-yi)*(yf-yi);
    if(Math.sqrt(x+y)+vi*1.5>r1+r2){
        console.log("N")
    } else{
        console.log("Y")
    }
}
